service/datasets/example.yaml

```diff
@@ -2,16 +2,16 @@ Database:
   example:
     example:
       rows: 1
-      A:
+      a:
         type: int
         is_key: True
         min: 0
-        max: 5
-      B:
+        max: 50
+      b:
         type: int
         min: 0
-        max: 25
-      C:
+        max: 20
+      c:
         type: int
         min: 0
-        max: 125
+        max: 1
```

Comment:
Pretty sure these conflicts with `externs_ml` will go away when we push my PR, so let's just keep in mind to do that first and I think we won't have to worry about them.

Reply:
Sounds good.